.github/workflows/binary-compatibility.yml

```diff
@@ -1,6 +1,12 @@
 name: Binary Compatibility
 
-on: [push, pull_request]
+on:
+  pull_request:
+    paths:
+      - 'assertj-core/**'
+  push:
+    paths:
+      - 'assertj-core/**'
 
 jobs:
   verify-with-base:
```

Comment:
The binary compatibility check with the `main` branch fails because it tries to look already at the new path.